### ISSUE TITLE
solver.prototxt for training?

### ISSUE THREAD

**Issue by dingtiandu** [anhttran/3dmm_cnn#12]
Can you share your solver.prototxt for training?Thanks.

**Comment by wanggongziZeo** [comment]
Do you have the file now?

**Comment by iacopomasi** [comment]
Hi,
we are not planning to release the training configuration file now but we might do it later on.
You can use the `deploy.prototxt` and use it for training: just adjust the Batch Normalization configuration and add L2 loss function. More advanced loss functions can be implemented using caffe layers.

Thank you.
